### PR TITLE
loc repo fallback to en-us for JoinTOC

### PR DESCRIPTION
**Pull request by AngryBerryMS**
[AB#351594](https://dev.azure.com/ceapex/Engineering/_workitems/edit/351594)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6922)

### CODE REVIEW

**Review by yufeih — CHANGES_REQUESTED** [review]
Add a spec test to repro the problem

**Comment by AngryBerryMS** [comment]
> Add a spec test to repro the problem

Resolved.

File: docs/specs/localization.yml [hunk 1]
```diff
@@ -840,3 +840,53 @@ repos:
...
+repos:
+  https://github.com/user/loc-fallback#main:
+    - files:
+        .openpublishing.publish.config.json: |
+          {
+            "docsets_to_publish": [{  "build_source_folder": "a"}],
+            "JoinTOCPlugin": [{"ReferenceTOC": "a/api/bot/TOC.yml", "TopLevelTOC": "a/docs-ref-toc/top_level_toc.yml"}]
+          }
+        a/docfx.yml:
+        a/_themes/ContentTemplate/schemas/ReferenceContainer.schema.json: |
```
Comment (yufeih) [inline docs/specs/localization.yml:855]: remove this file?

Comment (AngryBerryMS) [reply docs/specs/localization.yml]: Removed.

File: docs/specs/localization.yml [hunk 2]
```diff
@@ -840,3 +840,34 @@ repos:
 outputs:
   .errors.log: |
     {"message_severity":"error","code":"fallback-error","message":"Error(s) from 'en-us' repository caused this build failure, please check 'en-us' build report."}
+---
+# loc repo fallback to en-us
+locale: zh-cn
+repos:
+  https://github.com/user/loc-fallback#main:
+    - files:
+        .openpublishing.publish.config.json: |
```
Comment (yufeih) [inline docs/specs/localization.yml:849]: also remove this file as the content duplicates with the one from loc repo

**Review by yufeih — APPROVED** [review]

**Pull request merged**